spec/basic_examples_spec.rb

```diff
@@ -4,16 +4,16 @@ def expect_failure(&block)
   expect(&block).to raise_exception(RSpec::Expectations::ExpectationNotMetError)
 end
 
-RSpec.describe 'some basic hypothesis tests' do
-  it 'should think integer addition is commutative' do
+RSpec.describe 'basic hypothesis tests' do
+  it 'think integer addition is commutative' do
```

Comment:
`it think`: your bad grammar tolerance is greater than mine, :smile:, of course I see the issue, `it` is intended for singular subjects, but here the subject is plural.  Rather than being forced to choose between grammatically correct stdout and code, why not alias `it` to `they`?

Reply:
I like it. Will do.